### PR TITLE
#441 enable jobs section and disable reg form

### LINKED ISSUE #441: Enable Jobs and disable Reg Form

### PR DESCRIPTION
**Pull request by SuelaCark**
Enabled jobs section and disabled the registration form. Closes #441

### CODE REVIEW

**Comment by asynchroza** [comment]
Missing hashtag in PR title

File: packages/web/src/components/admin_page/admin.css [hunk 1]
```diff
@@ -113,9 +113,10 @@
     flex-direction: row;
     flex-wrap: wrap;
     justify-content: center;
-    align-items: center;
-    width: 80vw;
-    margin-left: 10vw;
+    align-items: flex-start;
```
Comment (asynchroza) [inline packages/web/src/components/admin_page/admin.css:116]: what's the idea here?

Comment (SuelaCark) [reply packages/web/src/components/admin_page/admin.css:116]: The jobs card in the admin panel had align-items: center which made them look unaligned correctly due to each card having different text lengths (see image below). So the align-items: flex-start at least aligns the cards in the same row at the top. I did this for aesthetic purposes for people from other departments to understand the cards easier. You can also see the difference if you run this PR locally and check [thehub-aubg](https://thehub-aubg.com/admin/dashboard/jobs) for comparison. 

image

Comment (asynchroza) [reply packages/web/src/components/admin_page/admin.css:116]: cool

**Review by asynchroza — APPROVED** [review]
Let's merge this and wait for dev deployment. If all is good, I will cherrypick this.

**Pull request merged**